src/main/java/com/znsio/e2e/exceptions/VisualTestSetupException.java

```diff
@@ -0,0 +1,7 @@
+package com.znsio.e2e.exceptions;
+
+public class VisualTestSetupException extends RuntimeException{
+    public VisualTestSetupException(String message) {
+        super(message);
+    }
+}
```

Comment:
Add the 2nd constructor as well (similar to other custom exceptions)

Reply:
ok